### PR TITLE
added valid moves for knight

### PR DESCRIPTION
**Pull request by jsilvajr1**
added valid moves for knight chess piece

### CODE REVIEW

**Comment by msarit** [comment]
Looks good to me! 😄

File: app/models/knight.rb [hunk 1]
```diff
@@ -1,2 +1,16 @@
 class Knight < Piece
-end
+
+  def valid_move?(x,y)
+    return false unless super(x,y)
+    x_diff = x_diff(x)
```
Comment (ilyakrasnov) [inline app/models/knight.rb]: @jsilvajr1 is there a method `x_diff` and `y_diff` somewhere?  are you sure it works? or do you mean `coord_x - x` or similar?

would be great to add a few test cases here :)

Comment (ilyakrasnov) [reply app/models/knight.rb]: @jsilvajr1 any progress on this?

Comment (jsilvajr1) [reply app/models/knight.rb]: Hi! Ilya, can you check my code?

**Comment by Linda091** [comment]
What's the status on this PR?

**Comment by msarit** [comment]
I believe @ilyakrasnov needs to approve the most recent changes that @jsilvajr1 made. Then it's good to be merged 😄

**Comment by bryanjim** [comment]
+1 Merge

**Pull request merged**